src/components/Search/CompareTable.jsx

```diff
@@ -0,0 +1,151 @@
+import { styled } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import store from '../../common/store';
+import { deleteRevision } from '../../reducers/SelectedRevisions';
+
+const StyledTableCell = styled(TableCell)(() => ({
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 14,
+  },
+  '&:first-of-type': {
+    width: '50px',
+  },
+  paddingTop: '10px',
+  paddingBottom: '10px',
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  '&:nth-of-type(odd)': {
+    backgroundColor: theme.palette.action.hover,
+  },
+}));
+
+function CompareTable(props) {
+  let { revisions } = props;
+
+  const selectedRevisions = [];
+  revisions?.forEach((item) => {
+    if (selectedRevisions.length === 0) {
+      selectedRevisions.push({ ...item, type: 'base' });
+    } else {
+      selectedRevisions.push({ ...item, type: 'new' });
+    }
+  });
+  revisions = selectedRevisions;
+  return (
+    <TableContainer
+      component={Paper}
+      style={{
+        boxShadow: 'none',
+        paddingBottom: '50px',
+        display: 'flex',
+        flexDirection: 'column',
+      }}
+    >
+      <Table style={{ minWidth: 700 }} aria-label="customized table">
+        <TableHead>
+          <TableRow>
+            <StyledTableCell />
+            <StyledTableCell
+              align="left"
+              style={{
+                fontWeight: 600,
+              }}
+            >
+              Project
+            </StyledTableCell>
+            <StyledTableCell align="left" style={{ fontWeight: 600 }}>
+              Revision
+            </StyledTableCell>
+            <StyledTableCell align="left" style={{ fontWeight: 600 }}>
+              Author
+            </StyledTableCell>
+            <StyledTableCell align="center" style={{ fontWeight: 600 }}>
+              Commit message
+            </StyledTableCell>
+            <StyledTableCell align="left" style={{ fontWeight: 600 }}>
+              Timestamp
+            </StyledTableCell>
+            <StyledTableCell />
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {revisions.map((row) => (
+            <StyledTableRow key={row.id}>
+              <StyledTableCell component="th" scope="row">
+                <div
+                  style={{
+                    fontWeight: 600,
+                    justifyContent: 'center',
+                    borderRadius: '2.5px',
+                    paddingTop: '5px',
+                    paddingBottom: '5px',
+                    border: '1px solid #d3d3d3',
+                    display: 'flex',
+                    color: '#d3d3d3',
```

Comment:
We need a darker color for accessibility. You can use https://app.contrast-finder.org/?lang=en to find a good one, or maybe https://design.firefox.com/photon/visuals/color.html#grey as well.